opencog/moses/deme/deme_expander.h

```diff
@@ -45,7 +45,8 @@ struct deme_expander
 	              optimizer_base& opt,
 	              const deme_parameters& pa = deme_parameters(),
 	              const subsample_deme_filter_parameters& fp = subsample_deme_filter_parameters(),
-	              type_node output_t = id::boolean_type);
+	              type_node output_t = id::boolean_type,
+	              string_seq labels={});
```

Comment:
Note that you can still pass optional argument to a reference as long as it is `const`, so for instance here you could have

```suggestion
	              const string_seq& labels={});
```

I don't think it's gonna make a different in performance here, unless perhaps you have millions of input features, just something to consider.